plenum/common/messages/fields.py

```diff
@@ -264,7 +276,7 @@ def _specific_validation(self, val):
 
 class LedgerIdField(ChooseField):
```

Comment:
Make sure we have tests for this (including tests for PLUGIN_LEDGER_IDS)

Reply:
Added `test_plugin_ledger_ids`